src/components/filters.js

```diff
@@ -0,0 +1,28 @@
+const createFilter = function (filter, isChecked) {
```

Comment:
```suggestion
const createFilterTemplate = function (filter, isChecked) {
```
Давай пока что по шаблону *create[FileName]Template*